### PR TITLE
fix: dropdown direction when at the bottom of the page in menu

### LINKED ISSUE #3770: [ENG-159] Language popup shown off-screen
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

When the language selector is in the lower part of the screen, the menu pops up partly or entirely outside the browser screen

### Expected Behavior

The whole popup menu should be visible

### Steps To Reproduce

Click the language selector when it is in the lower part of the screen

### What platform(s) does this occur on?

Web

### What browser(s) does this occur on?

Chrome

### Anything else?

https://github.com/lensterxyz/lenster/assets/667227/1398d7ea-d14b-421a-bfb4-093575560419

<sub>[ENG-159](https://linear.app/heyxyz/issue/ENG-159)</sub>

### PR DESCRIPTION
**Pull request by karanvishesh**
## What does this PR do?

It fixes the visibility of dropdown when selecting language.

## Related issues

Fixes #3770 
## Type of change

- Bug fix (non-breaking change which fixes an issue)

/claim #3770

### CODE REVIEW

File: apps/web/src/components/Shared/Footer/Locale.tsx [hunk 1]
```diff
@@ -28,6 +30,13 @@ const Locale: FC = () => {
     location.reload();
   }, []);
 
+  useEffect(() => {
```
Comment (bigint) [inline apps/web/src/components/Shared/Footer/Locale.tsx:33]: dont use useeffect unless its a network or external call - dan abramov

Comment (karanvishesh) [reply apps/web/src/components/Shared/Footer/Locale.tsx]: Done.

**Comment by bigint** [comment]
@karanvishesh can we make it support for all menu in hey?

**Comment by bigint** [comment]
Link T-662

**Comment by karanvishesh** [comment]
> @karanvishesh can we make it support for all menu in hey?

Yes we can.

**Pull request closed without merging**